Samsung/a71n/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="config_mainBuiltInDisplayCutout"></string>
```

Comment:
I don't understand, A71 definitely has a cutout?

Reply:
I've checked the value. it is present in /vendor/overlay. looks like Samsung trick Android to think Infinity O as a cutout